ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-file-answer/activityFileAnswer.component.spec.ts

```diff
@@ -95,100 +95,99 @@ describe('ActivityFileAnswer', () => {
-        //     spyOn(component.valueChanged, 'emit');
-        //     modalDialogServiceSpy.getDialogConfig.and.returnValue({});
-        //     fileUploadServiceSpy.getUploadUrl.and.returnValue(of([{
-        //         uploadGuid: 'uploadGuid',
-        //         uploadUrl: 'uploadUrl'
-        //     }]));
-        //     fileUploadServiceSpy.uploadFile.and.returnValue(of({}));
-        //     component.uploadedFiles = [];
-        //     component.onFilesSelected([{name: 'fileName', size: 2000000, type: '*.png'}] as unknown as Event);
-        //     fixture.detectChanges();
-        //
-        //     expect(component.block.answer).toEqual([{fileName: 'fileName', fileSize: 5000000, fileMimeType: '*.png'}]);
-        //     expect(component.valueChanged.emit).toHaveBeenCalledWith(['uploadGuid']);
-        //     // expect(component.fileNameToUpload).toBe('');
-        //     expect(component.errorMessage).toBe('');
-        // });
-
-        // it('should show error message if a local validator failed', () => {
-        //     const errorMessage = 'The file size is too big';
-        //     const localValidator = new ActivityFileValidationRule({} as any);
-        //     localValidator.result = errorMessage;
-        //     localValidator.recalculate = () => false;
-        //     component.block.validators = [localValidator];
-        //
-        //     // component.submitFileUpload({} as File, 'uploadGuid', 'uploadUrl');
-        //     expect(component.errorMessage).toBe(errorMessage);
-        // });
+        it('should patch file answer after upload', () => {
+            spyOn(component.valueChanged, 'emit');
+            modalDialogServiceSpy.getDialogConfig.and.returnValue({});
+            fileUploadServiceSpy.getUploadUrl.and.returnValue(of([{
+                uploadGuid: 'uploadGuid',
+                uploadUrl: 'uploadUrl'
+            }]));
+            fileUploadServiceSpy.uploadFile.and.returnValue(of({}));
+            component.uploadedFiles = [];
+            component.onFilesSelected({target: {files: [{name: 'fileName', size: 5000000, type: '*.png'}]}} as unknown as Event);
+            fixture.detectChanges();
+
+            expect(component.block.answer[0])
+                .toEqual(jasmine.objectContaining({fileName: 'fileName', fileSize: 5000000, fileMimeType: '*.png'}));
+            expect(component.valueChanged.emit).toHaveBeenCalledWith(['uploadGuid']);
+            expect(component.errorMessage).toBe('');
+        });
+
+        it('should show error message if a local validator failed', () => {
+            const errorMessage = 'The file size is too big';
+            const localValidator = new ActivityFileValidationRule({} as any);
+            localValidator.result = errorMessage;
+            localValidator.recalculate = () => false;
+            component.block.validators = [localValidator];
+
+            component.validateFiles([{}] as File[]);
```

Comment:
Instead of ` component.validateFiles([{}] as File[]);` 
you can just call
```
 fileUploadServiceSpy.getUploadUrl.and.returnValue(of([{
                uploadGuid: 'uploadGuid',
                uploadUrl: 'uploadUrl'
            }]));
component.onFilesSelected(
  {target: {files: [{name: 'fileName', size: 5000000, type: '*.png'}]}} as unknown as Event
);
```  
like in the previous test.